README.md

```diff
@@ -159,12 +75,9 @@ Hub for a list of all the supported tags.
 
 The following ports are exposed by this container:
 
-| Port | Purpose        |
-|------|----------------|
-| 8080 | Example only; nothing is actually listening on the port |
-
-The sample [Docker composition](docker-compose.yml) publishes the
-exposed port at 8080.
+| Port | Purpose                                             |
+|------|-----------------------------------------------------|
+| 8080 | Rest API to manage tasks and trigger sending emails |
```

Comment:
Very minor change to improve readability - take it or leave it as you see fit:
```suggestion
| 8080 | Rest API to manage tasks and trigger the sending of emails |
```

Reply:
@jsf9k resolved with [8bd475c](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/8bd475c699b223b12b09dfa9503bc1cd7da6cc8c)